### DISCUSSION TITLE
manager/installation-and-setup/Quick-Installation-on-Oracle-Cloud/

### DISCUSSION THREAD

**Discussion by giscus** [hiddify/Hiddify-Manager#4577] [Content Help کمک به اصلاح محتوا]
# manager/installation-and-setup/Quick-Installation-on-Oracle-Cloud/

The proxy manager platform.

https://hiddify.com/manager/installation-and-setup/Quick-Installation-on-Oracle-Cloud/

**Comment by HappyPoly** [comment]
I have already finished installion, but the link is only a ip address. I click the link, then it shows 404

**Comment by HappyPoly** [comment]
The problem has been solved.

**Reply by aghamorad** [reply]
May I ask how it was solved?

**Reply by HappyPoly** [reply]
> May I ask how it was solved?

Need to release the corresponding port in the panel

**Comment by 90rk1** [comment]
...aaand there is nothing at this page
